.circleci/setup.sh

```diff
@@ -79,8 +79,8 @@ if ! type bioconda-utils 2> /dev/null || [[ $BOOTSTRAP == "true" ]]; then
     # step 3: install bioconda-utils
     additional_packages='git pip'
     if [[ $OSTYPE == darwin* ]]; then
-        # Pinned to 2.5.3 to make sure we don't get unexpected changes.
-        additional_packages="${additional_packages} conda-forge-ci-setup=2.5.3"
+        # Pinned conda-forge-ci-setup to a specific version to make sure we don't get unexpected changes.
+        additional_packages="${additional_packages} conda-forge-ci-setup=2.6.0"
```

Comment:
`2.5.3` and `2.6.0` should work the same apart from that https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/69 is included, which may come in handy if/when we support per-recipe SDK choices and have packages that need newer versions.